packages/matter.js/test/util/storage/StringifyToolsTest.ts

```diff
@@ -68,6 +68,19 @@ describe("JsonConverter", () => {
             assert.ok(decodedObj instanceof Uint8Array);
         });
 
+        it("encode/decode Node.js Buffer", () => {
+
+            const obj = { type: 'Buffer', data: [1, 2, 3] };
```

Comment:
But are you sure this already is a jsonified Buffer? I would have expected that a normal Buffer would come in here, does your code work then still?

Reply:
yeah, sorry, this was a left over of me using the test method for trying out various inputs, I'll have added back in the Buffer object that i intended to have here.